src/components/FAQLayout/index.tsx

```diff
@@ -5,23 +5,80 @@ import { toSlug } from '../../utils/slug'
 
 interface Props {
   title: string
-  questions: string
+  gettingStartedQuestions: string
+  generalQuestions: string
+  bestPracticesQuestions: string
+  specificationQuestions: string
+  frontendQuestions: string
   rawMarkdownBody: string
 }
 
-const index = ({ title, questions, rawMarkdownBody }: Props) => {
+const index = ({ title, gettingStartedQuestions, generalQuestions, bestPracticesQuestions, specificationQuestions, frontendQuestions, rawMarkdownBody }: Props) => {
   return (
     <section>
       <div className="documentationContent">
         <div className="inner-content">
           <h1>{title}</h1>
-          {questions && (
+          {gettingStartedQuestions && (
```

Comment:
@carolstran You don't need a guard here and below since these sections should be always present

Reply:
This is updated in #985 😊